package.json

```diff
@@ -46,8 +46,6 @@
     "ember-infinity": "0.2.3",
     "ember-load-initializers": "0.5.1",
     "ember-resolver": "2.0.3",
```

Comment:
I removed these lines after realizing that my initial experiments didn't use sinon after all... but it's not clear that anyone else's tests require these dependencies either. Should I keep these lines in for now?